pkg/network/tracer/tracer_linux_test.go

```diff
@@ -468,7 +467,7 @@ func (s *TracerSuite) TestConntrackDelays() {
 
 	// The random port is necessary to avoid flakiness in the test. Running the the test multiple
 	// times can fail if binding to the same port since Conntrack might not emit NEW events for the same tuple
-	port := 5430 + rand.Intn(100)
+	port := 0
```

Comment:
Is `port` used anywhere else other than below?

Reply:
I don't believe so. It's only used when creating the TCP server.